src/components/NurseryWithdrawals/NurseryWithdrawalsDetails.tsx

```diff
@@ -174,7 +174,7 @@ export default function NurseryWithdrawalsDetails({
             justifyContent='space-between'
             alignItems='center'
           >
-            <Typography color={theme.palette.TwClrTxt} fontSize='20px' fontWeight={600} fontStyle='italic'>
+            <Typography color={theme.palette.ClrTextFill} fontSize='24px' lineHeight='32px' fontWeight={600}>
```

Comment:
This is a token from the old theme. Do we want to use it here?

Reply:
We don't have this color in the new theme. I added a comment in figma asking the new theme to be updated. We have a few other such usages in our codebase.

Reply:
see https://www.figma.com/file/yRSfer1xtWKAJr4IETPml1?node-id=2074:504340#330623402

Reply:
Thanks!

Reply:
I will create a ticket to track removal of old theme.